students/synowiec_krzysztof/lesson_07_string_datetime/to_swap_the_two_words.py

```diff
@@ -0,0 +1,7 @@
+def main():
+    first, second = input().split()
+    print("{0} {1}".format(first, second))
```

Comment:
This does not swap the words, they are printed exactly the same as input.
@synulewar please fix.